src/config.js

```diff
@@ -1,53 +1,95 @@
 import fs from 'fs';
 import merge from 'merge-deep';
 import yaml from 'js-yaml';
+import * as path from 'path';
 
-function getConfig(options) {
+let warningShown = false;
+
+export function getConfig(options) {
   let config = {};
   let { configPath } = options;
   if (!configPath) {
     configPath = `${process.cwd()}/.openapi-cli.yaml`;
 
-    if (fs.existsSync(`${process.cwd()}/.openapi-cli.yaml`)) {
-      configPath = `${process.cwd()}/.openapi-cli.yaml`;
-    }
-
-    if (fs.existsSync(`${process.cwd()}/.openapi-cli.yml`)) {
-      configPath = `${process.cwd()}/.openapi-cli.yml`;
+    if (fs.existsSync('.redocly.yaml')) {
+      configPath = path.resolve('.redocly.yaml');
+    } else if (fs.existsSync('.redocly.yml')) {
+      configPath = path.resolve('.redocly.yml');
+    } else if (fs.existsSync('.openapi-cli.yaml')) {
+      if (!warningShown) process.stderr.write('warning: .openapi-cli.yaml is deprecated, rename to .redocly.yaml\n');
```

Comment:
This misses that the actual structure of the file has changed too -- it should have `lint:`

Reply:
Nevermind, I see you also have that message below.